### PR TITLE
sio-tactician in process

### PR DESCRIPTION
**Pull request by MokhaLeee**
https://decomp.me/scratch/WAdU2

### CODE REVIEW

File: src/sio_tactician.c [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+        Proc_Break(proc);
+    }
+    else
+    {
+        SioPlaySoundEffect(0);
+    }
+}
+
+#if 0
+bool sub_8044B78(struct ProcTactician * proc, const struct TacticianTextConf * conf, u32 c, int d)
```
Comment (RevoSucks) [inline src/sio_tactician.c:272]: Is there a decomp.me scratch for this one?

**Comment by jiangzhengwenjz** [comment]
https://decomp.me/scratch/AwOTY solved

File: src/sio_tactician.c (hunk 1 above)
Comment (MokhaLeee) [reply src/sio_tactician.c:271]: Oh I forgot to put on it, now matched by @jiangzhengwenjz

**Comment by MokhaLeee** [comment]
updated nomatching code

**Pull request merged**